### PR TITLE
make UI changes to Watch Expressions Component (#1491)

### PR DESCRIPTION
**Pull request by jbhoosreddy**
Associated Issue: #1491

### Summary of Changes

* removed the "+" button
* improve style for light and dark themes
* improved watch expressions UI

### CODE REVIEW

**Comment by jasonLaster** [comment]
CC @clarkbw Jaideep and I talked about the UI and agreed that persisting the select box was a *simpler* design and perhaps a little cleaner.

**Comment by clarkbw** [comment]
👍 it only takes up one extra line and makes it much simpler in general.

File: src/actions/pause.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import type { Pause, Frame, Expression, Grip } from "../types";
 import type { ThunkArgs } from "./types";
 
 type CommandType = { type: string };
+type frameIdType = string | null;
```
Comment (wldcordeiro) [inline src/actions/pause.js]: Could we do this as a maybe? `string?`

Comment (jasonLaster) [inline src/actions/pause.js]: Hmm, i don't think so actually. `null` has special meaning for the frameId i believe. So it's not like checking for the presence of frameTypeId.

**Pull request merged**